manylinux1-wheel-build/Makefile

```diff
@@ -0,0 +1,46 @@
+WD = $(shell pwd)
+USERNAME := $(shell docker info | grep Username | xargs | cut -d ' ' -f 2)
+TARGET := $(notdir $(WD))
+ROOT := $(abspath $(WD)/../Pillow)
+IMAGENAME := $(if $(USERNAME), $(USERNAME)/$(TARGET), $(TARGET))
+BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
+TEST_TARGET = ubuntu-20.04-focal-amd64
+TEST_IMAGE=$(if $(USERNAME), $(USERNAME)/$(TEST_TARGET), $(TEST_TARGET))
+.PHONY: build
+build:
+	docker build -t $(IMAGENAME):$(BRANCH) .
+
+.PHONY: update
+update:
+	./update.sh
+
+.PHONY: wheel
+wheel:
+	-[ ! -d out ] && mkdir out
+	docker run --rm -v $(ROOT):/Pillow -v `pwd`/out:/output $(IMAGENAME):$(BRANCH) /build.sh $(_PYVER)
+
+36:
+	_PYVER=36 $(MAKE) wheel
+37:
+	_PYVER=37 $(MAKE) wheel
+38:
+	_PYVER=38 $(MAKE) wheel
+39:
+	_PYVER=39 $(MAKE) wheel
+
+.PHONY: test
+test: 38
+	docker run --rm -v $(ROOT):/Pillow -v `pwd`/out:/output $(TEST_IMAGE):$(BRANCH) sh -c "/vpy3/bin/pip install /output/*cp38-manylinux1_x86_64.whl && cd /Pillow && /vpy3/bin/python selftest.py && /usr/bin/xvfb-run -a /vpy3/bin/pytest -vx"
```

Comment:
What is the thinking behind using 38 rather than 39? It would useful to understand this for the future.

Reply:
Because that's what ubuntu 18.04 and 20.04 are using, and where I started with this. I threw the others in for completeness.